### ISSUE TITLE
Calculating Softmax over a sequence of scalars

### ISSUE THREAD

**Issue by coopie** [tensorflow/fold#56]
As the title suggests, I cannot find a way of doing this. There does not seem to be a way of currying a divide function with the sum of a sequence, and then mapping that back over the sequence.

Any help would be great - but I fear I will have to retreat to vanilla TF and use padded input sequences (the horror!).
